gtsfm/two_view_estimator.py

```diff
@@ -26,6 +27,40 @@
 pil_logger.setLevel(logging.INFO)
 
 
+@dataclass(frozen=False)
+class TwoViewEstimationReport:
```

Comment:
So it is in the other PR. But since we import it in `cycle_consistency.py`, we need to include it here, while we wait for the other PR to be ready for merge